COLLADAMax/src/COLLADAMaxColladaPlugin.cpp

```diff
@@ -16,10 +16,7 @@ namespace COLLADAMax
 
 	const TCHAR* COLLADAPlugin::PLUGIN_VERSION_TCHAR = __T("Version: 1.4.1");
 	const String COLLADAPlugin::PLUGIN_VERSION_STRING = "Version: 1.4.1";
```

Comment:
This is the COLLADA Schema version, not to be mixed up with he version of the plugin.